src/test/java/com/commercetools/sync/commons/asserts/statistics/CustomObjectSyncStatisticsAssert.java

```diff
@@ -0,0 +1,14 @@
+package com.commercetools.sync.commons.asserts.statistics;
+
+import com.commercetools.sync.customobjects.helpers.CustomObjectSyncStatistics;
+
+
+import javax.annotation.Nullable;
+
+
+public final class CustomObjectSyncStatisticsAssert extends
+        AbstractSyncStatisticsAssert<CustomObjectSyncStatisticsAssert, CustomObjectSyncStatistics> {
+    CustomObjectSyncStatisticsAssert(@Nullable final CustomObjectSyncStatistics actual) {
```

Comment:
leave an empty line between class definition and constructor

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/569/commits/a8c7ce4c770a85be1efb2f658ea00f9ee8c58393